src/screens/QuizScreenNoAcctScreen.tsx

```diff
@@ -0,0 +1,337 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  Dimensions,
+  TouchableOpacity,
+} from 'react-native';
+import Icon from 'react-native-vector-icons/Entypo';
+import { RFValue } from 'react-native-responsive-fontsize';
+import CircularTimer from 'react-native-circular-timer';
+import { QuestionResult } from './QuizResultsScreenNoAcctScreen';
+import { OfflineContact } from './noAcctHelperFunctions';
+
+interface QuestionOption {
+  contact: OfflineContact;
+  isCorrect: boolean;
+}
+
+interface Props {
+  navigation: any;
+  route: {
+    params: {
+      contacts: OfflineContact[];
+      questionResults: QuestionResult[];
+      QuizTitleListName: string;
+      CurrentQuizQuestionNumber: number;
+    };
+  };
+}
+
+// interface State {
+//     menuVisible: boolean;
+//     searchQuery: string;
+//     contacts: OfflineContact[];
+//     questionResults: QuestionResult[];
+//     QuizTitleListName: string;
+//     CurrentQuizQuestionNumber: number;
+//     loading: boolean;
+//     refreshing: boolean;
+//   }
```

Comment:
Is there reason to keep this in commented out?

Reply:
Nope, I just forgot to remove it. I can do it sometime today